website/content/partials/ent-supported-storage.mdx

```diff
@@ -1,3 +1,4 @@
-For enterprise customers, HashiCorp provides official support for Consul and Vault's Integrated Storage as storage backends. Vault Enterprise will
-no longer startup if configured to use a storage backend other than Consul or Integrated Storage. This is meant to protect against issues caused by using
-unsupported backends that do not support transactional storage.
+For enterprise customers, HashiCorp provides official support for Vault's Integrated Storage and Consul as storage backends. Vault Enterprise customers are strongly
+recommended to use these supported storage backends for best outcomes. Version 1.12.0 of Vault Enterprise will not startup if configured to use a storage backend
```

Comment:
nit: should "startup" be "start up"?

Reply:
Yes, but I actually think it reads better if we just remove the word up. What do you think?

Reply:
Even better! 👍

Reply:
Thanks for the feedback :)